Dockerfile

```diff
@@ -11,7 +11,7 @@ ARG build_date
 ARG work_dir=/github.com/aws-controllers-k8s/$service_alias-controller
 WORKDIR $work_dir
 # For building Go Module required
-ENV GOPROXY=https://proxy.golang.org,direct
+ENV GOPROXY=direct
```

Comment:
Why? this is disabling the proxy

Reply:
Sorry, this keeps sneaking into my pull requests because I have to change it to run anything on my local system.